tasklist/importer-860/src/main/java/io/camunda/tasklist/zeebeimport/v860/processors/es/VariableZeebeRecordProcessorElasticSearch.java

```diff
@@ -76,29 +153,32 @@ private UpdateRequest persistVariable(
       entity.setValue(recordValue.getValue());
     }
     entity.setFullValue(recordValue.getValue());
-    return getVariableQuery(entity);
+    return entity;
   }
 
-  private UpdateRequest getVariableQuery(final VariableEntity entity) throws PersistenceException {
+  private UpdateRequest prepareUpdateRequest(
+      final VariableListViewEntity variableListViewEntity, final String routingKey)
+      throws PersistenceException {
     try {
-      LOGGER.debug("Variable instance for list view: id {}", entity.getId());
-      final Map<String, Object> updateFields = new HashMap<>();
-      updateFields.put(VariableIndex.VALUE, entity.getValue());
-      updateFields.put(VariableIndex.FULL_VALUE, entity.getFullValue());
-      updateFields.put(VariableIndex.IS_PREVIEW, entity.getIsPreview());
+      final UpdateRequest request =
+          new UpdateRequest()
+              .index(tasklistListViewTemplate.getFullQualifiedName())
+              .id(variableListViewEntity.getId())
+              .upsert(objectMapper.writeValueAsString(variableListViewEntity), XContentType.JSON)
+              .routing(routingKey)
```

Comment:
```suggestion
              .routing(variableListViewEntity.getScopeKey())
```
like that you can remove `routingKey` parameter from this method